CMakeLists.txt

```diff
@@ -34,6 +34,10 @@ include(CMakePackageConfigHelpers)
 
 set(TORCHVISION_CMAKECONFIG_INSTALL_DIR "share/cmake/TorchVision" CACHE STRING "install path for TorchVisionConfig.cmake")
 
+if(NOT CMAKE_INSTALL_LIBDIR)
+  set(CMAKE_INSTALL_LIBDIR lib)
+endif()
+
```

Comment:
I think you can remove this, it should be set when `GNUInstallDirs` is included on line 32.
Or does it fail without it?

Reply:
OK. I remove it.

Reply:
Done.